### ISSUE TITLE
[Issue] Unittest Fails when building in MacOS/darwin

### ISSUE THREAD

**Issue by mspi92** [regclient/regclient#297]
### Current Behavior

Checking out the Repo and running `make` gives this failure:

> time="2022-09-12T23:44:58+02:00" level=warning msg="Error running script" error="<string>:2: manifest pull failed: platform not found: darwin/arm64\nstack traceback:\n\t[G]: in function 'config'\n\t<string>:2: in main chunk\n\t[G]: ?" script=GetConfig
time="2022-09-12T23:44:58+02:00" level=info msg="Copy image" digestTags=false dry-run=false forceRecursive=false includeExternal=false script=CopyLatest source="ocidir://testrepo:v1" target="ocidir://testcopy:latest"
time="2022-09-12T23:44:58+02:00" level=info msg="Copy image" digestTags=false dry-run=false forceRecursive=false includeExternal=false script=DeleteCopy source="ocidir://testrepo:v1" target="ocidir://testdel:old"
time="2022-09-12T23:44:58+02:00" level=info msg="Delete tag" dry-run=false image="ocidir://testdel:old" script=DeleteCopy
time="2022-09-12T23:44:58+02:00" level=info msg="Copy image" digestTags=false dry-run=true forceRecursive=false includeExternal=false script=DryRun source="ocidir://testrepo:v1" target="ocidir://testdryrun:latest"
time="2022-09-12T23:44:58+02:00" level=warning msg="Error running script" error="<string>:3: Context error: context deadline exceeded\nstack traceback:\n\t[G]: in function 'ls'\n\t<string>:3: in main chunk\n\t[G]: ?" script=Timeout
--- FAIL: TestRegbot (0.02s)
    --- FAIL: TestRegbot/GetConfig (0.00s)
        regbot_test.go:167: unexpected error on process: failure in user script
FAIL
coverage: 25.0% of statements
FAIL    github.com/regclient/regclient/cmd/regbot       0.677s

### Expected Behavior

Unittests are working independent of GOOS.
**Additionally: afaik there are no docker/oci images with platforms `darwin/arm64` & `darwin/amd64`**

### Steps To Reproduce

On MacOS (darwin), without container:

```sh
git clone https://github.com/regclient/regclient.git
cd regclient
make
```

### Version

Happens on latest main

### Environment

- Running as binary or container: binary
- Host platform: Darwin/arm64
- Registry description: doesn't matter

**Comment by sudo-bmitch** [comment]
Thanks for the report. I believe this is fixed in #300 but I don't have a MacOS system to test on, so let me know if it's not the case.

**Comment by mspi92** [comment]
Yes, it works, thanks for the adjustment
